rule/blank-imports.go

```diff
@@ -52,14 +55,20 @@ func (w lintBlankImports) Visit(_ ast.Node) ast.Visitor {
 		if !isBlank(imp.Name) {
 			continue // Ignore non-blank imports.
 		}
+
 		if i > 0 {
 			prev := w.fileAst.Imports[i-1]
 			prevPos := w.file.ToPosition(prev.Pos())
-			if isBlank(prev.Name) && prevPos.Line+1 == pos.Line {
+
+			if isBlank(prev.Name) && prevPos.Line+1 == pos.Line && prev.Path.Value != embedImportPath {
```

Comment:
we could add a variable to document the semantics of this condition. Something like:
 ```suggestion
			isSubsequentBlancInAGroup := isBlank(prev.Name) && prevPos.Line+1 == pos.Line && prev.Path.Value != embedImportPath 
			if  isSubsequentBlancInAGroup {
```

Reply:
Sure, why not :)